app/src/main/kotlin/com/exallium/todoapp/editnote/EditNotePresenter.kt

```diff
@@ -61,47 +68,75 @@ class EditNotePresenter(view: EditNoteView,
 
             override fun onError(t: Throwable) {
                 Timber.w(t, "error looking up note for edit with id " + noteId)
-                view.showUnableToLoadNoteError()
+                view::showUnableToLoadNoteError
```

Comment:
did not realize that view::showUnableToLoadNoteError was not an equivalent to view.showUnableToLoadNoteError()!